### DISCUSSION TITLE
v0.21.3

### DISCUSSION THREAD

**Discussion by alex-w** [Stellarium/stellarium#2128] [General]
**The installers are located in the "Assets" section at the end of these release notes!**

[![Github Release](https://img.shields.io/github/downloads/Stellarium/stellarium/v0.21.3/total.svg)](https://github.com/Stellarium/stellarium/releases/v0.21.3)

A huge thanks to the people who helped us a lot by reporting bugs!

## Release notes
The Stellarium team is delighted to announce the release of Stellarium 0.21.3.

The major changes of this version:
- Better handling of minor bodies with outdated orbital elements
- "Goto next day at same altitude" functionality for selected objects
- Mark a few special points on the planets
- Three refined Arab skycultures
- Updated Mul-Apin skyculture with new artwork
- Improved OnlineQueries and Satellites plugin
- Fixed a problem with LX200 telescopes
- Many fixes in core and plugins

There has also been a large number of bug fixes and closed feature requests and enhancements (see full list of changes).

Enjoy!

## What's Changed
Full list of changes:
- Added support a context in names of sky cultures
- Added improvements for compute the accuracy and performance of sunrise/sunset time in AstroCalc/WUT
- Added support HTC 2.0 theory (#1942)
- Added also search descriptions in Satellites plugin (GH: #1940)
- Added support QtWebEngine in OnlineQueries plugin (GH: #1938)
- Added reference to ArchaeoLines plugin
- Added scriptable method to set the time extent for trails
- Added figure correction for the Lunar position
- Added atmospheric pressure and temperature info into location tooltip
- Added *.md files into Windows installer (GH: #2023)
- Added orbit epoch as explicit element to the KeplerOrbit (GH: #2000, #2027)
- Added Solar and Lunar info to InfoString (GH: #1455, #2037)
- Added pragmatic solution for a Linux display problem with aberration (GH: #1945)
- Added a "default" group to selected object info (GH: #2038)
- Added context for info group names (GH: #2038)
- Added epoch entries for the Interstellar objects
- Added support to show ephemeris when Solar system bodies are hidden (GH: #2044, #2051)
- Added orbit_Epoch to comet orbital elements
- Added scripting functions for tonemapping
- Added tool to find and select an object from its standard name and object type name (GH: #1970)
- Added tool to search an object by its standard name and object type
- Added Arabic sky culture (GH: #2064)
- Added Arabian Peninsula sky culture (GH: #2064)
- Added Arabic Lunar Stations sky culture (GH: #2064)
- Added new actions for jumping to defined moments in diurnal motion of selected objects (GH: #2106)
- Added smart tooltip for AstroCalc/Graphs tool (GH: #2110)
- Added Poles and east and west points (limb/equator intersection) (GH: #2092, #2098)
- Added Central and Subsolar special points (GH: #2098)
- Added scriptable method to replace textures on-the-fly (GH: #1060, #2120)
- Added twilight data into InfoString for the Sun
- Added new filter for satellites (GH: #2124)
- Added code to automatic add the group of satellites, based on Celestrak groups (GH: #1636, #2125)
- Fixed issues in hawaiian starlines sky culture (GH: #1931)
- Fixed error in search of Atlas right hand (GH: #1934)
- Fixed computation max elevation in AstroCalc/WUT tool
- Fixed building unit tests when scripting is disabled
- Fixed direct script commands for RemoteControl (GH: #1943)
- Fixed selecting sky culture
- Fixed Sun limb size and direction in Navigational Stars plugin (GH: #2019, #2021)
- Fixed Observability plugin: restore lost lunar "previous" and "next" full moon information (GH: #2025)
- Fixed epoch support for the Pluto (GH: #2062)
- Fixed OnlineQueries plugin: additional cmake fixes over #2006 (GH: #2049)
- Fixed a regular expression for constellation references (GH: #2055)
- Fixed typo in HD designation for HIP 48715 star (GH: #2063)
- Fixed dithering for native GLES2 systems (Odroid C1)
- Fixed PNGs: prevent libpng iCCP incorrect sRGB profile warnings
- Fixed building OnlineQueries plugin for non-WebEngine mode
- Fixed typo in CMake script (GH: #2067)
- Fixed default epoch for comet orbit
- Fixed crash after importing single asteroid (pulsar, quasar, supernova, nova, exoplanetary system) when imported asteroid was selected (GH: #1970)
- Fixed crash after updating catalogs of objects (pulsars, quasars, supernovae, novae, exoplanets, solar system objects) when selected object was removed by updating tool (GH: #1970)
- Fixed formatting (GH: #2095)
- Fixed rendering of markers when using perspective or orthographic projections (GH: #2093)
- Fixed issue in implementation of LX200 protocol (GH: #818)
- Fixed typos in sky cultures (GH: #2105)
- Fixed context issue for translation of pole labels (GH: #2098)
- Fixed aberration issues for Nomenclature selection (GH: #2098)
- Fixed selection priority for far-side features of the planet (GH: #2098)
- Changed AstroCalc/Transits tool: cosmetic fix for custom markers
- Changed Satellites plugin: improve guessing GALILEO satellites
- Changed core: refactoring the support of the on-screen stars names (GH: #1951, #2005)
- Changed the core: replace QRegExp occurrences
- Changed mouse navigation behaviour (GH: #1317, #2086, #2089)
- Updated minimum requirements: Qt 5.9
- Updated instructions for building Stellarium from source code (GH: #1927)
- Updated GUI of Satellites plugin: cosmetic fixes
- Updated list of exoplanet detection methods
- Updated list of contributors
- Updated list of active developers
- Updated planetary nomenclature
- Updated list of base locations
- Updated the code of core: refactoring
- Updated translations of sky cultures
- Updated translations of 3D sceneries
- Updated translations of landscapes
- Updated translations of GUI
- Updated the description of Chinese sky culture and fixed some issues (GH: #1952)
- Updated reference in Calendars plugin
- Updated pulsars catalog
- Updated documents for Scripting Engine
- Updated Stellarium User Guide
- Updated code for select priority and cosmetic fixes (GH: #2098)
- Updated code for tooltips in AstroCalc/Graphs tools (GH: #2110)
- Updated Babylonian (mulapin) sky culture
- Removed dead code
- Removed outdated code and data
- Removed Arabic Moon Stations sky culture (GH: #2064)

## Packages notes
package | note
------------|---------
stellarium-0.21.3.tar.gz | Stellarium 0.21.3 for UNIX/Linux (source code). <br />__MD5:__ 37dd6da287914fb8a6cfc865966479d3 <br />__SHA1:__ a3396c403050e073c592da695faeb048c1d5e19b <br />__SHA256:__ 8ac6c054d12f136fe0d5c0deaaa8d7b69dd2a462be1711a187364574aef97e7f
stellarium-0.21.3-win32.exe | Stellarium 0.21.3 for Windows (32-bit; Windows 7+) based on Qt 5.12.6 (Visual Studio 2017 Community) with media support. <br /> __MD5__: 65d07e78a862f13da439e070f197b283 <br />__SHA1__: 4445ebc15b9dd5964cf582f39d7a2fe30abe3020 <br />__SHA256__: 8467e2f7feffb22c100d6b18fc992aabc4885536d741f23d983174def31fca6d
stellarium-0.21.3-win64.exe | Stellarium 0.21.3 for Windows (64-bit; Windows 7+) based on Qt 5.12.6 (Visual Studio 2017 Community) with media support. <br /> __MD5__: 7e3cce3264e42567f7e6a2e32791dda9 <br />__SHA1__: 8386e16ebb0ac27d01afe036144599d7dfd95a81 <br />__SHA256__: 6d4279e4ec13368115fe0c0acb0b862f856b7b1d9560dc07f0791c44937873a6
Stellarium-0.21.3.zip | Stellarium 0.21.3 for Mac OS X (64-bit; Mac OS X 10.12+) based on Qt 5.12.9 (AppleClang 11.0.0) with media support. <br /> __MD5__: 1e7e80bf8fbabb06e38cb6495d456b9c <br />__SHA1__: 32be44e917f535defcb1267b5d07624b08c54f57 <br />__SHA256__: d6c7eb5770ec53366d73297742a18feda800e4efad664afe2600efc851b3589c
Stellarium-0.21.3-x86_64.AppImage | Stellarium 0.21.3 for Linux (AMD64; glibc 2.27) based on Qt 5.9.5 (GCC 7.5.0) with media support. <br /> __MD5__: acf2800c7ae65d503f8cd0175bff23f7 <br />__SHA1__: 216d0072ed08ceac3da5eff2749aca17618a904a <br />__SHA256__: c1fed2f6efa94e0dc72e2080cdffc2defe85806e82be63d463e22dc2d711e05e

<hr /><em>This discussion was created from the release <a href='https://github.com/Stellarium/stellarium/releases/tag/v0.21.3'>v0.21.3</a>.</em>

**Comment by NotSoRandomOne** [comment]
Congratulations on the update, you guys!
>"Goto next day at same altitude" functionality for selected objects...
Searching the new help PDF (and the interface), I cannot see how this is activated. Sorry if it is obvious and I missed it, but if I did not miss it because it isn't there, you may want to update that PDF.

**Reply by alex-w** [reply]
Please check shortcuts editor ;)

**Reply by gzotti** [reply]
The User Guide section 4.8.1 describes the process with similar commands. The principle applies also here. But we are running out of memorable keys, so you should only assign the variants you are most likely going to use.

**Reply by NotSoRandomOne** [reply]
Thanks!

**Comment by PwrHsePro** [comment]
What is asp and the ApImage in the download section?

**Reply by gzotti** [reply]
The only "asp" on this page is in your question. What do you mean?
AppImage: Why not Google for it?

**Comment by PwrHsePro** [comment]
Instalarium, I clicked on update it took me to a page where at the bottom there were different updates available for whatever operating system you had.. There are Various executables where one of them says ASP and 1 of them says Something about image. There are updates there from 21. 1 to go to 21. 3Clear Skies!Jeff
-------- Original message --------From: Georg Zotti ***@***.***> Date: 2/5/22  2:18 PM  (GMT-06:00) To: Stellarium/stellarium ***@***.***> Cc: PwrHsePro ***@***.***>, Comment ***@***.***> Subject: Re: [Stellarium/stellarium] v0.21.3 (Discussion #2128) 
The only "asp" on this page is in your question. What do you mean?
AppImage: Why not Google for it?

**Reply by gzotti** [reply]
ASC (not asp) files are tiny files which allow those in the know to verify their downloads have not been tampered with.

**Comment by PwrHsePro** [comment]
Cool... tyWhat is the other one thats called Image something?Clear Skies!Jeff
-------- Original message --------From: Georg Zotti ***@***.***> Date: 2/5/22  6:41 PM  (GMT-06:00) To: Stellarium/stellarium ***@***.***> Cc: PwrHsePro ***@***.***>, Comment ***@***.***> Subject: Re: [Stellarium/stellarium] v0.21.3 (Discussion #2128) 
ASC (not asp) files are tiny files which allow those in the know to verify their downloads have not been tampered with.

**Reply by gzotti** [reply]
Follow my previous suggestion.

**Comment by PwrHsePro** [comment]
Well... i dont see a suggestion... I asked what 2 of the 3 downloads were in thw updates where I downloaded the exe for V21.3... your first response aksed where I was seeing it... your second response explained 1 of the files... but I never did see an explanatuon of the third file.I am not at my PC and cant duplicate the location via my smart phone... The link I used to get the update came from the Stellarium 'About' page in the program on my PCClear Skies!Jeff
-------- Original message --------From: Georg Zotti ***@***.***> Date: 2/6/22  5:30 AM  (GMT-06:00) To: Stellarium/stellarium ***@***.***> Cc: PwrHsePro ***@***.***>, Comment ***@***.***> Subject: Re: [Stellarium/stellarium] v0.21.3 (Discussion #2128) 
Follow my previous suggestion.

**Reply by gzotti** [reply]
I wrote
> AppImage: Why not Google for it?

If I don't know a "computer term", I look into Wikipedia or Google for an explanation.

**Comment by PwrHsePro** [comment]
I generally Google stuff... BUT... there is a forum for questions and I wanted to know what it was specifically to Stellarium.

I am not a neophyte when it comes to computers... not a tecnician... but not a neubie either.

Ive used Stellarium for many years and have set it up for many others to use... on the 2 PCs I personally own and use I have exe's going back quite a ways... see pic...

![20220206_091650](https://user-images.githubusercontent.com/90519146/152689733-2cb427e1-7e1b-46da-91e6-6330ae6de908.jpg)

Ive set up and controlled dozens of mounts with Stellarium including Orion/SkyWatcher, Meade, Cel3stron, Astrophysics, and 10 Micron.

I apologize if my 2 questions cause you discomfort.

Clear Skies!
Jeff

**Comment by ultrapre** [comment]
Hello, when would v0.22.0 release?
